neo/core/__init__.py

```diff
@@ -37,6 +37,7 @@
 from neo.core.analogsignal import AnalogSignal
 from neo.core.irregularlysampledsignal import IrregularlySampledSignal
 
+# Import FilterClasses
```

Comment:
This imports the FilterClasses into to main namespace, e.g. class `Equal` is a very general name and could possibly shadow other names and cause unintended effects.